tools/local-network/README.md

```diff
@@ -1,14 +1,25 @@
 ## Intro
 
-This directory contains shell scripts for running a local 5-node mesh network and a `mobilecoind` instance.
+This directory contains scripts for running a local mobilecoin consensus network and a `mobilecoind` instance.
+The nodes are run in debug mode, so it is only intended to be used for development and testing purposes.
+
+To run a local network, you will provide both the `IAS_API_KEY` and `IAS_SPID`, which you can obtain by registering with the [Intel SGX Portal](https://api.portal.trustedservices.intel.com/EPID-attestation
 
 ## bootstrap.sh
 
+Each node in the network requires a ledger containing at least one block to start. Further, to perform transactions on the network you will
+need private master keys.
+
 This is a simple script to bootstrap a test ledger and keys to play with. The script writes the keys to `target/sample_data/keys` and the ledger to `target_sample_data/ledger`.
 
-## local-network-5.sh
+## local-network.py
+
+This script starts a local mobilecoin consensus network by launching a separate process for each consensus validator and configuring them to communicate via a default set of ports. It takes the following parameters:
+
+- (required) `--network-type` - describes the network topology, one of `dense5`, `a-b-c`, `ring5` or `ring5b`
+- (optional) `--skip-build` - does not rebuild consensus node binaries
 
-This script starts a 5 node network configured in a mesh topology. It relies on environment variables for configuration:
+It relies on environment variables for configuration:
```

Comment:
There are some changes to the env vars.
- `CLIENT_RESPONDER_HOST` no longer exists.
- `MOB_RELEASE` defaults to 1, might be worth mentioning.
- `LEDGER_SYNC_S3` no longer exists.